src/components/common/CustomSelect.vue

```diff
@@ -1,14 +1,32 @@
 <template>
-    <div :tabindex="tabindex" class="custom-select" @blur="open = false">
-        <span class="icon fw-light text-muted">
-            <img src="../../assets/icons/git-network-outline-muted.svg" class="fw-light text-muted" style="width: 16px">
+    <div
+        :tabindex="tabindex"
+        class="custom-select"
+        :style="
+            hasBorder
+                ? { border: 'solid 1px #dee2e6', borderRadius: '0.25rem' }
+                : ''
+        "
+        @blur="open = false"
+    >
+        <span v-if="icon" class="icon fw-light text-muted">
+            <img
+                :src="require(`@/assets/icons/${icon}`)"
+                class="fw-light text-muted"
+                style="width: 16px"
+            />
         </span>
         <div
             :class="{ open: open }"
             class="selected fw-light text-muted"
+            :style="!icon ? { paddingLeft: '12px' } : ''"
             @click="onSelectClick"
         >
-            In {{ selected?.displayValue }}
+            {{
+                hasInPrefix
```

Comment:
@sosan-neikbeen I would make this more generic. Like ```hasPrefix``` and then define the prefix ```prefix="In"``` as component prop. ```hasInPrefix``` reads weird.